pkg/resourcemonitor/resourcemonitor.go

```diff
@@ -259,6 +257,28 @@ func (rm *resourceMonitor) updateNodeCapacity() error {
 	return nil
 }
 
+func (rm *resourceMonitor) resUpdated(old, new interface{}) {
```

Comment:
at first I was a bit skeptical towards this approach, but the more I read about it the more I like it. Good idea, let's try this.